apps/contracts/defindex/src/test/admin.rs

```diff
@@ -0,0 +1,116 @@
+use soroban_sdk::{testutils::{Address as _, AuthorizedFunction, AuthorizedInvocation}, Address, IntoVal, Symbol, Vec, vec as sorobanvec};
+
+use crate::test::{DeFindexVaultTest, create_strategy_params};
+
+extern crate alloc;
+use alloc::vec;
+
+#[test]
+fn test_set_new_fee_receiver_by_fee_receiver() {
+    let test = DeFindexVaultTest::setup();
+    let strategy_params = create_strategy_params(&test);
+    let tokens: Vec<Address> = sorobanvec![&test.env, test.token0.address.clone(), test.token1.address.clone()];
+    let ratios: Vec<u32> = sorobanvec![&test.env, 1, 1];
+
+    test.defindex_contract.initialize(&test.emergency_manager, &test.fee_receiver, &test.manager, &tokens, &ratios, &strategy_params);
+
+    let fee_receiver_role = test.defindex_contract.get_fee_receiver();
+    assert_eq!(fee_receiver_role, test.fee_receiver);
+
+    let users = DeFindexVaultTest::generate_random_users(&test.env, 1);
+    // Fee Receiver is setting the new fee receiver
+    test.defindex_contract.set_fee_receiver(&test.fee_receiver, &users[0]);
```

Comment:
You are mocking all auths for all tests.
Instead:
1.- mock all auths only for tests that you really dont need to check aauths
2.- for tests that you really need to check auths, mock the specific authorizaion, like in
https://github.com/soroswap/core/blob/11021feefd2b6e160dc9cf5770b21e4336ebaaa9/contracts/factory/src/test/fee_to_setter.rs#L30



3 Check previous authorizations
https://github.com/soroswap/core/blob/11021feefd2b6e160dc9cf5770b21e4336ebaaa9/contracts/factory/src/test/fee_to_setter.rs#L46